app/views/votes/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Vote</h1>
+<br>
+<br>
+<button type = "button" id = "hotbut" class = ".btn-lg" >HOT</button>
```

Comment:
please no spaces around the `=` in html attributes
`<button type="button" id="hotbut" class=".btn-lg" `